src/referentielijsten/api/tests/test_tabellen_admin.py

```diff
@@ -0,0 +1,44 @@
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from referentielijsten.accounts.tests.factories import UserFactory
+
+from .factories import TabelFactory
+
+
+@disable_admin_mfa()
+class TabellenAdminTests(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.superuser = UserFactory.create(superuser=True)
+
+    def test_items_link_returns_correct_admin_url(self):
+        tabel = TabelFactory.create()
+
+        # TODO replace with reverse(..., query=...) once Django is upgraded to 5.2
+        expected_url = reverse("admin:api_item_changelist") + f"?tabel__id={tabel.id}"
+        expected_html = (
+            f'<a href="{expected_url}">{_("Show items in item list view")}</a>'
+        )
+
+        response = self.app.get(
+            reverse("admin:api_tabel_changelist"), user=self.superuser
+        )
+        self.assertIn(expected_html, response.content.decode())
+
+    def test_items_link_target_url_is_accessible(self):
+        tabel = TabelFactory.create()
+
+        # TODO replace with reverse(..., query=...) once Django is upgraded to 5.2
+        url = reverse("admin:api_item_changelist") + f"?tabel__id={tabel.id}"
+
+        response = self.app.get(url, user=self.superuser)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(str(tabel.id), response.text)
```

Comment:
I think this test would be a bit more robust if you:
- assert that the changeview URL of the table is in `response.text`, currently it just checks if the `id` (which is just a number) is in the page
- create a different tabel with the factory and explicitly assert that the URL for that table is not in `response.text`, that way we know the filter is working as expected